tests/examples/test_examples.py

```diff
@@ -198,7 +201,9 @@ def _assert_snapshot(example, url, example_type):
 
     assert success, "%s failed to load" % example.relpath
     assert no_resources, "%s failed with %d missing resources" % (example.relpath, len(resources))
-    assert no_errors, "%s failed with %d errors" % (example.relpath, len(errors))
+
+    if example_type != "server":
```

Comment:
@canavandl, I will look into this. It's likely that a simple generalization of the existing algorithm should be sufficient, though I have to look into when server creates documents.